src/WaxSigningApi.ts

```diff
@@ -210,12 +210,14 @@ export class WaxSigningApi {
     const controller = new AbortController();
 
     setTimeout(() => controller.abort(), 5000);
-
+    const waxjsVersion = version || null;
+    
     const response: any = await fetch(`${this.waxAutoSigningURL}signing`, {
       body: JSON.stringify({
         freeBandwidth: !noModify,
         feeFallback,
         transaction: Object.values(serializedTransaction),
+        waxjsVersion
```

Comment:
I don't think we need to create the waxjsVersion since it only gets used here, so
```
waxjsVersion: version
```